pyresample/test/test_spatial_mp.py

```diff
@@ -19,11 +19,7 @@
 """Testing the _spatial_mp module."""
 
 
-# try:
-#     from unittest import mock
-# except ImportError:
-#     # separate mock package py<3.3
-#     import mock
+# from unittest import mock
```

Comment:
Why leave this as a comment if it isn't used ?

Reply:
I think it is used in the code below that is also commented out. Since I don't know why the test code was commented out I thought I'd leave this here.